tests.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import flask
+import os
```

Comment:
Are we using all those imports?

Reply:
No, not at this time, I removed them and recommitted.